Containerfile

```diff
@@ -0,0 +1,30 @@
+FROM opensuse/tumbleweed:latest
+
+# Install dependencies
+RUN zypper in -y -C \
+    git \
+    python3-osc \
+    python3-openqa_client \
+    python3-pika \
+    python3-requests \
+    python3-ruamel.yaml \
+    python3-jsonschema
```

Comment:
```suggestion
    python3-jsonschema && \
    zypper clean
```

Reply:
Actually, better do that after the next zypper call

Reply:
True. And I guess I can probably merge the two installation in one zypper install call as this is not intended to be build by OBS anyway.